packages/ember/lib/index.js

```diff
@@ -115,7 +115,7 @@ Ember.unwatchPath = metal.unwatchPath;
 Ember.watch = metal.watch;
 Ember.isWatching = metal.isWatching;
 Ember.unwatch = metal.unwatch;
-Ember.destroy = metal.destroy;
+Ember.destroy = metal.deleteMeta;
```

Comment:
could rename here too not sure

Reply:
No, we need to keep `Ember.destroy` for now.